girder/utility/install.py

```diff
@@ -138,8 +139,12 @@ def runWebBuild(wd=None, dev=False, npm='npm', allPlugins=False, plugins=None, p
     :type plugins: str, list or None
     :param progress: A progress context for reporting output of the tasks.
     :type progress: ``girder.utility.progress.ProgressContext`` or None
+    :param coreOnly: Enable this to build the girder web with no additional plugins.
+    :type coreOnly: bool
     """
-    if isinstance(plugins, six.string_types) and plugins:
+    if coreOnly:
+        plugins = ['']
```

Comment:
Let's change the conditional on line 74 of this file to `if plugins is None`, that way we can set this to be `[]` and have it build no plugins.